index.html

```diff
@@ -1913,6 +1919,64 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    [=People=] should be provided relevant information when data is accessed or when permission is
+    requested.
+  </span>
+</div>
+
+Transparency is a necessary, but insufficient, condition for [=consent=]. Relevant information
+includes who may be accessing information, what information may be accessed (including the potential
+inferences or combinations of such data) and how information may be used. For transparency to be
+meaningful to people, it must be contextual.
```

Comment:
I worry about all the "mays." I know that this is a legalistic approach, but it tends to lead to bigger bags than necessary and participates in making people believe that a lot of their data needs to be accessed. I'm not sure how to capture that, but I think that transparency would be helped if it were more definitive.